ossdbtoolsservice/language/metadata_executor.py

```diff
@@ -90,7 +90,7 @@ def _columns(self, kinds=('r', 'v', 'm')):
         :return: list of (schema_name, relation_name, column_name, column_type) tuples
         """
 
-        if self.conn.server_version >= 80400:
+        if self.conn.connection.server_version >= 80400:
```

Comment:
Is this query PG specific? What about the versions?

Reply:
It is, I have moved this code to a separate PG  file.